runtime/compiler/runtime/SymbolValidationManager.cpp

```diff
@@ -88,6 +92,21 @@ TR::SymbolValidationManager::defineGuaranteedID(void *symbol, TR::SymbolType typ
    _seenSymbolsSet.insert(symbol);
    }
 
+bool
+TR::SymbolValidationManager::isClassWorthRemembering(TR_OpaqueClassBlock *clazz)
+   {
+   if (!_jlthrowable)
+      _jlthrowable = _fej9->getSystemClassFromClassName(jlthrowableName, (int32_t)strlen(jlthrowableName));
+
+   if (_jlthrowable &&  VM_VMHelpers::isSameOrSuperclass((J9Class *)_jlthrowable, (J9Class*)clazz))
```

Comment:
I think this heuristic deserves a comment in the code - it is not clear or obvious why one would want to do this.